src/pages/home/home.ts

```diff
@@ -70,14 +72,20 @@ export class HomePage {
     private kafka: KafkaService
   ) {}
 
+  ionViewWillEnter() {
+    this.getElementsAttributes()
+    this.elProgressHeight += 15
+    this.applyTransformations()
+  }
+
   ionViewDidLoad() {
     // const isFirstIonDidViewLoad = this.navParams.data.isFirstIonDidViewLoad
     this.checkForNextTask()
     this.evalHasClinicalTasks()
     this.checkIfOnlyESM()
 
     setInterval(() => {
-      this.isNextTaskESMandNotNow()
+      // this.isNextTaskESMandNotNow()
```

Comment:
Why is this removed ? I think this was to ensure that the task bar shows `Soon` if the next task is ESM. Has that changed or being handled elsewhere?